etlman/projects/authorizers.py

```diff
@@ -6,7 +6,7 @@ def user_is_authenticated(request: HttpRequest, **view_kwargs: dict) -> bool:
 
 
 def user_is_project_collaborator(request: HttpRequest, **view_kwargs: dict) -> bool:
-    project_id = view_kwargs["project_id"]
+    project_id = str(view_kwargs["project_id"])
```

Comment:
Why is this needed?

Reply:
This is due to a mypy complaint

`etlman/projects/authorizers.py:12: error: Incompatible type for lookup 'pk': (got "Dict[Any, Any]", expected "Union[str, int]")`

So it is expecting a int or str, but is getting a dict[any, any]